test/nn/encoder/test_stype_encoder.py

```diff
@@ -421,11 +406,23 @@ def test_text_tokenized_encoder():
         stype=stype.text_tokenized,
         col_to_model_cfg=col_to_model_cfg,
     )
-    feat_text = tensor_frame.feat_dict[stype.text_tokenized]
+    feat_text = copy.deepcopy(tensor_frame.feat_dict[stype.text_tokenized])
```

Comment:
Does `text_tokenized` do not use `MultiNestedTensor` yet?

Reply:
`text_tokenized` use a dictionary of `MultiEmbeddingTensor`, so `deepcopy` is necessary here.